ci/rails-lint.yml

```diff
@@ -0,0 +1,32 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# This workflow will download a prebuilt Ruby version, install dependencies, and  run linters
+name: Build Rails and run linters
```

Comment:
👋  There's no "Building" for Rails, should this just be "Run linters"?

Reply:
@joshmgross Thanks for the inputs. I will raise another PR to incorporate these.

Reply:
@joshmgross Addressed your inputs in this PR -> https://github.com/actions/starter-workflows/pull/1086